install_files/ansible-base/roles/build-securedrop-app-code-deb-pkg/files/rules-focal

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/make -f
+
+DEB_DH_INSTALL_ARGS=-X .git
+
+SECUREDROP_BUILD_PLATFORM=$(shell lsb_release -sc)
+
+%:
+	dh $@ --with python-virtualenv --with systemd
+
+override_dh_gencontrol:
+	dh_gencontrol -- $(SUBSTVARS)
+
+# Move the conffile in version control to squash the autogenerated one
+# by debhelper, as files in /etc/ are automatically marked as
+# conffiles.  We are shipping AppArmor profiles via this package, and
+# want them to be correctly updated with each update.
+override_dh_installdeb:
+	dh_installdeb
+	cp ${CURDIR}/debian/conffiles ${CURDIR}/debian/securedrop-app-code/DEBIAN/
+
+override_dh_strip_nondeterminism:
+	find ./debian/ -type f -name '*.pyc' -delete
+	find ./debian/ -type f -name 'pip-selfcheck.json' -delete
+	find -type f -name RECORD -exec sed -i -e '/.*\.pyc.*/d' {} +
+	dh_strip_nondeterminism $@
+
+override_dh_virtualenv:
+	dh_virtualenv \
+		--python=/usr/bin/python3.8 \
+		--extra-pip-arg "--verbose" \
```

Comment:
The only changes I can find in the rules file between xenial & focal are:

1. python3.5 vs. python3.8
2. xenial contains `--setuptools`, whereas focal does not

Ideally we'd have this file templated so the rest is the same, but for now, can you add a comment explaining why setuptools is required for xenial but not for focal?

Reply:
I did and then remembered why i don't need it. By default it will install latest `setuptools` and with `--setuptools` flag we can now specify a specific version.

Reply:
> By default it will install

Does "it" refer to the `dh-virtualenv` package? Even if the `--setuptools` flag is truly not required for the build process—and I agree, it doesn't appear to be—we still have an entire file duplicated except for this one flag, so a comment should be added to explain why it's not necessary.